requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask
```

Comment:
https://github.com/xeon-zolt/meilix-generator/compare/c5bb81cb...49703a9c
The reason behind removing these lines of equality?